src/pages/Clients.tsx

```diff
@@ -0,0 +1,55 @@
+import AppContainer from "../components/layout/AppContainer";
+import { Flex, Button, Input } from "@mantine/core";
+import { INPUTS_CLIENTS_PAGE } from "../constants/clients/clients";
+import CustomInput from "../components/shared/CustomInput";
+import { IconSearch, IconX } from "@tabler/icons-react";
+import CustomTable from "../components/shared/CustomTable";
+
+import {
+  CustomMainTitle,
+  ClientsAddingPanel,
+  ClientsInputsGroup,
+} from "../styles/ClientsPage";
+
+const Clients = () => {
+  return (
+    <AppContainer>
+      <CustomMainTitle order={1}>Клиенты</CustomMainTitle>
+
+      <ClientsAddingPanel>
+        <Flex gap="md" align="center">
+          <div>Всего: x</div>
+          <Button variant="filled">Добавить клиента</Button>
+        </Flex>
+      </ClientsAddingPanel>
+
+      <ClientsInputsGroup>
+        <Flex gap="md" align="center">
+          {INPUTS_CLIENTS_PAGE.map((input, index) => {
+            return (
+              <CustomInput
+                key={index}
+                label={input.label}
+                placeholder={input.placeholder}
+              />
+            );
+          })}
```

Comment:
```suggestion
        {INPUTS_CLIENTS_PAGE.map((input, index) =>
          <CustomInput
            key={index}
            label={input.label}
            placeholder={input.placeholder}
          />
        )}
```

do not use index for key